controls/pcidss_4_ocp4.yml

```diff
@@ -475,9 +564,13 @@ controls:
         - Any other security-related wireless vendor defaults.
       levels:
         - base
-      status: pending
+      status: not applicable
```

Comment:
Should this be automated, or partial, since we have at least one rule here checking that wireless interfaces are disabled?

Reply:
Aha - good call. Thanks!